webapp/store/auth.js

```diff
@@ -84,6 +84,7 @@ export const actions = {
             role
             about
             locationName
+            locale
             contributionsCount
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-31T10:24:45Z" title="Thursday, October 31st 2019, 11:24:45 am +01:00">Oct 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -46,6 +46,9 @@ export const getters = {
   termsAndConditionsAgreed(state) {
     return state.user && state.user.termsAndConditionsAgreedVersion === VERSION
   },
+  locale(state) {
+    return state.locale
```


should we have a fallback here @ogerly ? most of the getters I have seen in or store have `|| null`, etc...

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-31T15:26:43Z" title="Thursday, October 31st 2019, 4:26:43 pm +01:00">Oct 31, 2019</time>_

---


@ogerly I think you can remove this. You're consuming `this.user.locale` so this code here is never used.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-31T15:30:59Z" title="Thursday, October 31st 2019, 4:30:59 pm +01:00">Oct 31, 2019</time>_

---


I agree with @mattwr18 that we should have a fallback. However, since you're not using this, I would suggest to have a fallback where you use it:
```js
        this.$i18n.set(this.user.locale || 'en')
```